assets/js/modules/analytics/datastore/__fixtures__/settings--default.json

```diff
@@ -1,15 +1,15 @@
 {
-  "ownerID": 0,
```

Comment:
The default owner ID should not change as this fixture is for the default settings as would be returned by `{Module}\Settings::get_default`

Reply:
Yikes - I thought these "fixtures" were used solely for tests!

Reply:
Yikes indeed, good to know!